### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by vegaro**
Adds a workflow that:

1. Exports the unity package (we can use this for deploys in the future)
1. Adds the package to a project called IntegrationTests that has dependency on Purchases
1. Exports the Android and iOS project from the IntegrationTests Unity project
1. Assembles both Android and iOS projects

### CODE REVIEW

**Comment by vegaro** [comment]
For the record. It looks like UnityTests are getting stuck even in my local machine

File: scripts/create-unity-package.sh [hunk 1]
```diff
@@ -34,31 +42,42 @@ if [ -d "$PROJECT/Assets/ExternalDependencyManager" ]; then
...
+    -exportPackage $FOLDERS_TO_EXPORT $PACKAGE
+fi 
+
+# if [[ $? -ne 0 ]]; then
+#   echo "😱 Exporting has failed! Checkout exportlog.txt"
+#   echo "😱 Unity shouldn't be running when executing the script."
+#   exit $return_code
+# else
+#   echo "🚀 Exporting finished! Path: $PACKAGE" 
+# fi
```
Comment (vegaro) [inline scripts/create-unity-package.sh:81]: this should be removed

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,230 @@
+version: 2.1
```
Comment (vegaro) [inline .circleci/config.yml:1]: Main changes here. Most of the other changes are in a new Unity project inside `IntegrationTests`

File: IntegrationTests/fastlane/Matchfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+git_url(ENV["CERTS_REPO_URL"])
+
+storage_mode("git")
+
+type("development")
+ 
+app_identifier(["com.revenuecat.UnityIntegration"])
+username(ENV["FASTLANE_USER"])
```
Comment (aboedo) [inline IntegrationTests/fastlane/Matchfile:8]: 👍

**Review by aboedo — APPROVED** [review]
great work on this!

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+          name: Bring scripts back
+          command: |
+            mv IntegrationTests/Assets/Editor/CIEditorScript.cs.break IntegrationTests/Assets/Editor/CIEditorScript.cs
+            mv IntegrationTests/Assets/Main.cs.break IntegrationTests/Assets/Main.cs
+
+  activate-license:
+    description: "Activates Unity license"
+    steps:
+      - run: echo $BASE64_LICENSE | base64 --decode >> Unity_v2019.x.ulf
+      - run: |
```
[the remark this answers is not in the document]
Comment (vegaro) [reply .circleci/config.yml:52]: totally agree lol that looks crazy

**Pull request merged**